### PR TITLE
[cli] Fix tests directories names

### PR DESCRIPTION
**Pull request by ferrarafer**
Rename tests directories to avoid redundancy.

- Changed services tests directory from `test/services_tests` to `test/service`
- Changed viewmodels tests directory from `test/viewmodel_tests` to `test/viewmodel`
- Changed utils tests directory from `test/util_tests` to `test/util`
- Updated stacked_tools tests, example and playground to align with directory names changes

This PR closes #601, was messy 🙉

### CODE REVIEW

File: packages/stacked_tools/lib/src/templates/compiled_templates.dart [hunk 1]
```diff
@@ -988,16 +1062,11 @@ void main() {
 
 // --------------------------------------------------
 
-// -------- Generic View Folder Template Path ----------
-
-const String kViewTemplateGenericFolderPath = 'lib/ui/views/generic/';
```
Comment (Eimen2018) [inline packages/stacked_tools/lib/src/templates/compiled_templates.dart:993]: @ferrarafer I added this for delete command i needed the folder path to remove the view and viewmodel

**Review by Eimen2018 — CHANGES_REQUESTED** [review]
Delete command would fail to get the right path... and i don't even know how you didn't see an error on delete_view_command.dart

**Comment by FilledStacks** [comment]
@ferrarafer I'm lost with this PR haha. I don't know if the changes I chatted to you about was applied. Send me a message on slack lets chat through it.

**Comment by FilledStacks** [comment]
@ferrarafer we can close this? The config updates should include the general fix for this problem.

**Comment by ferrarafer** [comment]
> @ferrarafer we can close this? The config updates should include the general fix for this problem.

Sure.

**Pull request closed without merging**